include/net/nrf_cloud_rest.h

```diff
@@ -297,6 +297,34 @@ int nrf_cloud_rest_disconnect(struct nrf_cloud_rest_context *const rest_ctx);
  */
 int nrf_cloud_rest_jitp(const sec_tag_t nrf_cloud_sec_tag);
 
+/**
+ * @brief Send a JSON formatted message using the SendDeviceMessage endpoint. See the
+ * nRF Cloud REST API documentation for proper JSON formatting.
+ *
+ * @param[in,out] rest_ctx Context for communicating with nRF Cloud's REST API.
+ * @param[in]     device_id Null-terminated, unique device ID registered with nRF Cloud.
+ * @param[in]     json_msg Null-terminated JSON string containing the device message.
+ * @param[in]     topic Optional; null-terminated MQTT topic string.
+ *
+ * @retval 0 If successful.
+ *          Otherwise, a (negative) error code is returned.
+ */
+int nrf_cloud_rest_send_device_message(struct nrf_cloud_rest_context *const rest_ctx,
+	const char *const device_id, const char *const json_msg, const char *const topic);
+
+/**
+ * @brief Send a supported NMEA sentence to nRF Cloud as a device message.
+ *
+ * @param[in,out] rest_ctx Context for communicating with nRF Cloud's REST API.
+ * @param[in]     device_id Null-terminated, unique device ID registered with nRF Cloud.
+ * @param[in]     nmea_sentence Null-terminated NMEA sentence.
```

Comment:
It could also be nice to have the option to send PVT data, but I guess there needs to be cloud-side support first

Reply:
nrf cloud supports the nmea sentences listed here:
https://github.com/nRFCloud/application-protocols/tree/v1/schemas/deviceToCloud/gps